src/randoop/main/GenInputsAbstract.java

```diff
@@ -219,20 +184,18 @@ public GenInputsAbstract(String command, String pitch,
   @OptionGroup("Values used in tests")
   @Option("Use null as an input with the given frequency")
   public static double null_ratio = 0;
-  
+
   /**
-   * Never use null as input to methods or constructors.
+   * Suppress use of null when no other argument value can be generated.
    * 
-   * This option causes Randoop to abandon the method call rather than providing
-   * null as an input, when no non-null value of the appropriate type is available.
+   * If true, Randoop will fail input generation rather than use null as an 
+   * input.
    * 
-   * To ask Randoop to calls methods with null with greater frequency,
-   * see option --null-ratio.
+   * Does not affect the behavior based on {@link #null_ratio}.
```

Comment:
Can you make the relationship between the two options more explicit?
Also, I see you removed the reference to --null-ratio.  I think you should reinstate it.  This text appears in the Randoop manual (many more people will read it there than in the source code), and it should make sense there using concepts that a Randoop user is familiar with.  Right now it appears in the manual as a confusing literal `#null_ratio`.
Other uses should be fixed too, such as the reference to --forbid-null from --null-ratio.